iphone/Classes/TiUIListViewProxy.m

```diff
@@ -498,7 +498,39 @@ -(void)setContentInsets:(id)args
         [self.listView setContentInsets_:arg1 withObject:arg2];
     }, [NSThread isMainThread]);
 }
+- (NSMutableArray *)getSelectedRows:(id)args
+{
+    NSMutableArray *result=[[NSMutableArray alloc]init];
```

Comment:
I think it's `ENSURE_ARG_COUNT(unused, 0)` -> Ensure that there are no args provided.